### PR TITLE
fix some typos in docs

### PR DESCRIPTION
**Pull request by tim-2**
### Summary
fix some typos in docs

### CODE REVIEW

File: lib/Mojolicious/Guides/Cookbook.pod [hunk 1]
```diff
@@ -255,7 +255,7 @@ And then we are going to need a C<Dockerfile> describing the container. A very s
 
 It uses the latest L<Perl container|https://hub.docker.com/_/perl> from Docker Hub, copies all the contents of your
 application directory into the container, installs CPAN dependencies with L<App::cpanminus>, and then starts the
-application on port C<3000> with the pre-forking web server. With L<Mojolicious::Command::generate::dockerfile> there is
+application on port C<3000> with the pre-forking web server. With L<Mojolicious::Command::Author::generate::dockerfile> there is
```
Comment (kraih) [inline lib/Mojolicious/Guides/Cookbook.pod:258]: Is that line the correct length?

**Comment by kraih** [comment]
Four commits for a few typo fixes is rather excessive.

**Comment by tim-2** [comment]
yeah sorry I messed it up.

**Comment by kraih** [comment]
Thanks anyway, i've fixed the typos since you closed the PR. https://github.com/mojolicious/mojo/compare/bd3a50d4f2c5...f13adfa79989

**Pull request closed without merging**